src/rest-server/src/routes/v2/virtual-cluster.js

```diff
@@ -41,11 +37,11 @@ router.route('/:virtualClusterName')
   .delete(token.check, controller.remove);
 
 router.route('/:virtualClusterName/status')
-  /** PUT /api/v2/virtual-clusters/:virtualClusterName - Change virtual cluster status (running or stopped) */
+  /** PUT /api/v2/virtual-clusters/:virtualClusterName/status - Change virtual cluster status (running or stopped) */
   .put(token.check, param.validate(vcConfig.vcStatusPutInputSchema), controller.updateStatus);
 
-router.route('/:virtualClusterName/resourceUnits')
-  /** GET /api/v2/virtual-clusters/:virtualClusterName/resourceUnits - Get virtual cluster available resourceUnits */
+router.route('/:virtualClusterName/units')
```

Comment:
>units [](start = 35, length = 5)

why rename this? but keep controller.getResourceUnits

Reply:
just renamed the api, current api uses dash `-` but `resourceUnits` is a camel style
the code uses camel case so the variable name is fine

Reply:
So why not resource-units

Reply:
updated